src/network/networkd-manager.c

```diff
@@ -690,6 +694,11 @@ int manager_start(Manager *m) {
 
         assert(m);
 
+        sysctl_add_monitor(m);
```

Comment:
it's ok to ignore return values here, but please cast the call to `(void)` to indicate that we ignore the return value here on purpose, not by accident. various static checkers do care about this hint.